common/projectconfig/projectconfig.go

```diff
@@ -56,6 +58,9 @@ func LoadConfig(ctx context.Context, input []string) (Config, error) {
 	if err != nil {
 		return Config{}, err
 	}
+	if config.FTLMinVersion != "" && !ftl.IsVersionAtLeastMin(ftl.Version, config.FTLMinVersion) {
+		return config, fmt.Errorf("FTL version '%v' predates the minimum version '%v'", ftl.Version, config.FTLMinVersion)
```

Comment:
Use `%q` to quote a string - it's simpler, but also it will correctly escape the string.

Reply:
Oh sweet thanks!